apps/vast/vast_threatbus/message_mapping.py

```diff
@@ -152,9 +151,9 @@ def matcher_result_to_sighting(matcher_result: str) -> Union[Sighting, None]:
             return None
         ref = ref[len(THREATBUS_REFERENCE) + 1 : -1]
     if ref is None:
-        ref = f"note--00000000-0000-0000-0000-000000000000"
-    context = {}
-    context["source"] = "VAST"
+        # All zeroes is no valid UUIDv4, at least these two bits must be set.
+        ref = f"note--00000000-0000-4000-8000-000000000000"
```

Comment:
It's not clear from the comment why this particular value was chose, but as long as it's documented somewhere, this works.

Reply:
I've reworded for clarity.